app/controllers/rubygems_controller.rb

```diff
@@ -20,6 +20,7 @@ def index
   def show
     @latest_version = @rubygem.versions.most_recent
     @versions       = @rubygem.public_versions(5)
+    render :show_all_yanked if @rubygem.public_versions.count.zero?
```

Comment:
It's good idea.

> Does that mean that gem was deleted?

It just means all versions are yanked. The owner(s) can push new versions yet.